lib/sms/client.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+module SMS
+  class Client
+    account_sid = 'ACf24246b793b0d591cff6522cbaf9d0d9'
+    auth_token = '19c4e05ec5e042828fbc3c895cee7054'
+    somo_phone_number = '437-800-2780'
```

Comment:
Probably consider extracting these constants to a config file and perhaps encrypting them with a hash